packages/@okta/vuepress-site/docs/guides/registration-inline-hook/activate-and-enable/index.md

```diff
@@ -20,15 +20,52 @@ The Registration Inline Hook is now set up with a status of active.
 
 > **Note:** You can also setup an inline hook using an API. See [Inline Hooks Management API](/docs/reference/api/inline-hooks/#create-inline-hook) for further information.
 
+### Enabling the Registration Inline Hook
+
+The procedure to enable the Registration Inline Hook is dependent on the type of org you are using: Okta Identity Engine or Okta Classic. Follow the procedure for your specific org.
+
+After you enable the Registration Hook, you are ready to preview and test the example.
+
+#### Enabling the Registration Inline Hook in Okta Identity Engine
+
+<ApiLifecycle access="ie" />
+If you have an Okta Identity Engine org, you must [Enable and configure a profile enrollment policy](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/create-profile-enrollment-policy-sr.htm) to implement a Registration Inline Hook.
+
+> **Note:** Profile Enrollment and Registration Inline Hooks only work with the [Okta Sign-In Widget](/code/javascript/okta_sign-in_widget/) version 4.5 or later.
+
+To associate the Registration Inline Hook with your Profile Enrollment policy:
+
+1. In the Admin Console, go to **Security > Profile Enrollment**.
+
+1. Click the pencil icon to edit the policy and associate it with your Registration Inline Hook.
+
+1. In **Enrollment Settings**, click the More Options icon and then select **Edit**. If not already selected, click **Allowed** in the **For new users** section.
```

Comment:
- "More Options icon" is capitalized, but the "pencil icon" (in the previous line) is not. I'm not sure what the convention is for referencing icons, but capitalization should be consistent?
- "click **Allowed**" => "select the **Allowed** check box" ?

Reply:
This whole page is ugly, unfriendly UX, imo. You need to know that the inverted ellipsis is the "More Options" icon. Anyway, I capitalized "Pencil" icon, and reformulated the "Allowed" radio button selection as: "Select **Allowed** for **Sign-up** in the **For new users** section."